test/Microsoft.ML.Tests/OnnxConversionTest.cs

```diff
@@ -1631,7 +1631,7 @@ public void OptionalColumnOnnxTest(DataKind dataKind)
             Done();
         }
 
-        [Fact]
+        [NativeDependencyFact("lib_lightgbm")]
```

Comment:
What about `[LightGBMFact]` ?

Reply:
So your point about the if is what I will do. In fact, we already skip it if we aren't on x64, so I just need to change the current `if` to include a check for arm/arm64.

Reply:
I made this change for this specific case, but I realized that many of the other tests in this specific file are doing something similar. I will refactor this file to do that for the other tests as well.

Reply:
Refactored those few tests as well into the `if` format you suggested.